modules/custom_operations/user_ie_extensions/tokenizer/tensorflow_translators.cpp

```diff
@@ -197,18 +178,9 @@ ov::OutputVector translate_reshape(const ov::frontend::NodeContext& node) {
     FRONT_END_GENERAL_CHECK(node.get_input_size() == 2, "Tensorflow Reshape op should have two inputs");
     auto tensor = node.get_input(0);
     auto shape = node.get_input(1);
-    if(auto pack = dynamic_cast<StringTensorPack*>(tensor.get_node())) {
-        // TODO: If it is a beginning of the graph, how to detect strings? It falls in 'else' branch in this case.
-        // FIXME: Needs extension for a Parameter to prepare it first
-        auto begins = std::make_shared<Reshape>(pack->input_value(0), shape, false);
-        auto ends = std::make_shared<Reshape>(pack->input_value(1), shape, false);
-        auto chars = pack->input_value(2);
-        auto reshape = post_translate_string_tensor_output({begins, ends, chars});
-        return {reshape};
-    } else {
-        auto reshape = std::make_shared<Reshape>(tensor, shape, false);
-        return {reshape};
-    }
+    auto reshape = std::make_shared<Reshape>(tensor, shape, false);
+    return {reshape};
+//    }
     // set_node_name(node.get_name(), reshape); // TODO: requires dependencies from TF FE internals
```

Comment:
do we need this? Please clean the code a bit.

Reply:
Deleted